lib/muchos/existing.py

```diff
@@ -72,6 +72,8 @@ def sync(self):
                 print("- import_playbook: mesos.yml", file=site_file)
             if config.has_service("swarmmanager"):
                 print("- import_playbook: docker.yml", file=site_file)
+            if config.has_service("elkserver"):
```

Comment:
Could you add additional validation here, checking if the `swarmmanager` role has also been assigned if `elkserver` has been assigned? It would help avoid user error.

Reply:
Please see previous message about this topic.